scripts/winesapos-setup.sh

```diff
@@ -309,6 +309,43 @@ if [ $? -eq 0 ]; then
     sudo swapon /swap/swapfile
     echo "/swap/swapfile    none    swap    defaults    0 0" | sudo tee -a /etc/fstab
     qdbus ${kdialog_dbus} /ProgressDialog org.kde.kdialog.ProgressDialog.close
+elif [ $? -eq 1 ]; then
+    # If the user does not want swap, ask them about zram instead.
+    kdialog --title "winesapOS First-Time Setup" --yesno "Do you want to setup zram instead (recommended if you do not use swap)?"
+    if [ $? -eq 0 ]; then
+        kdialog_dbus=$(kdialog --title "winesapOS First-Time Setup" --progressbar "Please wait for zram to be enabled..." 2 | cut -d" " -f1)
+        qdbus ${kdialog_dbus} /ProgressDialog Set org.kde.kdialog.ProgressDialog value 1
+        # zram half the size of RAM.
+        winesap_ram_size=$(free -m | grep Mem | awk '{print $2}')
+        zram_size=$(expr ${winesap_ram_size} / 2)
+        sudo touch /etc/systemd/system/winesapos-zram.service /usr/local/bin/winesapos-zram-setup.sh
+        
+        # Setup script to run on boot.
+        # Yes they are supposed to not be tabbed in.
+        echo """#!/bin/bash
+
+/usr/bin/modprobe zram
+echo ${zram_size}M > /sys/block/zram0/disksize
+echo zstd > /sys/block/zram0/comp_algorithm
```

Comment:
I did some testing with this. I found that the compression type needs to be defined before creating the zram device. Basically just switch lines 328 and 329 (`echo zstd` first and then `echo ${zram_size}M`).

Otherwise, it fails to set the compression type.

```
$ cat /sys/block/zram0/comp_algorithm
lzo [lzo-rle] lz4 lz4hc 842zstd
$ sudo systemctl status winesapos-zram.service | grep error
Sep 30 16:15:18 winesapos bash[4249]: /usr/local/bin/winesapos-zram-setup.sh: line 5: echo: write error: Device or resource busy
$ echo zstd | sudo tee /sys/block/zram0/comp_algorithm
-bash: echo: write error: Device or resource busy
```

Reply:
Alright, I fixed the order, should be OK now